mrblib/aws-s3.rb

```diff
@@ -2,32 +2,25 @@ module AWS
   class S3
     EMPTY_STRING_SHA256 = Digest::SHA256::hexdigest('')
 
-    def initialize(access_key, secret_key, security_token = nil, region = nil, endpoint = nil)
+    def initialize(access_key: nil, secret_key: nil, security_token: nil, region: nil, bucket_name: nil)
```

Comment:
Was keyword argument supported in current mruby?
If not use `Hash` object instead.